src/sentry/issues/utils.py

```diff
@@ -36,8 +41,12 @@ def can_create_group(
 
 def write_occurrence_to_platform(performance_problem: PerformanceProblem, project: Project) -> bool:
     return bool(
-        # handle only N+1 db query detector first
-        performance_problem.type.type_id == PerformanceNPlusOneGroupType.type_id
+        # handle only N+1 db query and consecutive http detector first
+        performance_problem.type.type_id
+        in [
+            PerformanceNPlusOneGroupType.type_id,
+            PerformanceConsecutiveHTTPQueriesGroupType.type_id,
+        ]
```

Comment:
you could remove these two checks because they'll be handled in https://github.com/getsentry/sentry/pull/46879/files#diff-0daa3c681e3c72bcdaa6f9ada27dfb4f973e134a2417ac0bc139277be86a392fR23

Reply:
Reverted the change.